streams/src/test/java/org/apache/kafka/streams/state/internals/CachingWindowStoreTest.java

```diff
@@ -339,6 +344,7 @@ public void shouldForwardDirtyItemsWhenFlushCalled() {
         assertNull(cacheListener.forwarded.get(windowedKey).oldValue);
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Likewise, it seems like we can migrate away from the deprecated method in this test.
(This is also true of the following methods where I didn't make a comment)

Reply:
removed